### ISSUE TITLE
add 1.16 blocks to main branch

### ISSUE THREAD

**Issue by IncredibleHolg** [overviewer/Minecraft-Overviewer#1807]
@CounterPillow I am opening this isse to add the missing blocks to the main branch. The code discussed in #1772 looks not yet stable, so adding the missing blocks in the ``classical`` way may get usable results faster than incorporating the modernGLRender branch.

Would you like to squash/merge up to a3e4812ca906146408171b34e3c313fcb5e7ee69  into the master branch before moving on?

Missing blocks:
* Nylium, Hyphae, shroomlight  and wart blocks
* All new logs
* All new wood types, new craftable wood blocks
* New craftable stone types

**Comment by CounterPillow** [comment]
Please just send a PR to the 116-blocks branch.

**Comment by IncredibleHolg** [comment]
Starting with the easy ones, warped and crimson nylium
![nylium-01](https://user-images.githubusercontent.com/41533138/87243192-38e38600-c434-11ea-8cb5-157392b6a048.png)
![nylium-02](https://user-images.githubusercontent.com/41533138/87243195-3da83a00-c434-11ea-9cab-06a4462b7c6d.png)

**Comment by IncredibleHolg** [comment]
Added logs aka stems
![nether_stems01](https://user-images.githubusercontent.com/41533138/87248535-34ca5f00-c45a-11ea-8262-f0625c87ae19.png)

![nether_stems02](https://user-images.githubusercontent.com/41533138/87248541-3ac04000-c45a-11ea-8dc7-ee79705d1854.png)

**Comment by IncredibleHolg** [comment]
Added hyphae
![nether_stems03](https://user-images.githubusercontent.com/41533138/87249553-05b6ec00-c460-11ea-9419-69437a210e82.png)

**Comment by IncredibleHolg** [comment]
crimson and warped biomes blocks are implemented PR #1812
![netherbiomes01](https://user-images.githubusercontent.com/41533138/87729004-9a08b200-c7c4-11ea-8fd7-595f27052a25.png)

**Comment by LeadMagnet** [comment]
I'm having issue with Polished Blackstone Bricks.

[What it looks like in game](https://photos.app.goo.gl/to9b5JtgjXD8iCT26)

[What it looks like after Overviewer render](http://map.texter.family/worlds/#/-85/64/-211/max/world%20-%20overworld/world)

Running version 0.15.88 on W2019 server.

**Comment by IncredibleHolg** [comment]
No surprise, they are not yet implemented

**Comment by LeadMagnet** [comment]
> No surprise, they are not yet implemented

AH...well that explains it :P

**Comment by IncredibleHolg** [comment]
Added blocks needed to clear the nether roof, see #1806 
Sha1: 0a9d196

**Comment by IncredibleHolg** [comment]
Added stone nether blocks , PR #1832 

![nether_stones_01](https://user-images.githubusercontent.com/41533138/89740742-cd6cf280-da8b-11ea-9209-133e35ac41b0.png)

![nether_stones_02](https://user-images.githubusercontent.com/41533138/89740744-d2ca3d00-da8b-11ea-915e-0f06b87b59cc.png)

**Comment by IncredibleHolg** [comment]
@LieuweWestra Has added missing wood blocks
![netherwood_1](https://user-images.githubusercontent.com/41533138/90299015-b87dce00-de94-11ea-84b7-7e0bfae56871.png)
![netherwood_2](https://user-images.githubusercontent.com/41533138/90299024-bd428200-de94-11ea-84fa-3405f13b7498.png)

**Comment by IncredibleHolg** [comment]
PR #1835 implements 
* crying_obsidian, lodestone, respawn_anchor
* ancient_debris part of the nether roof

this should fix #1831 and #1833 

If you want to test it, use ( assuming your remote is named origin )
```bash
git fetch origin pull/1835/head:testnether
git checkout testnether
```
sha1: 619ce0a

**Comment by drnixon** [comment]
Soul torches and soul lanterns are also failing to render in 0.16.3, nor do they get treated as light sources in nighttime renders. Soul campfires seem to show up fine, but I don't see soul fire in any soul soil valleys in my nether render.

**Comment by ProfDrBielefeld** [comment]
Are these still in a testbranch? Cause the latest version via apt cant render chains and soulfire as well as blackstone walls

**Comment by RossosHEX** [comment]
91ea874

no rendering of 
* Warped Wood Slabs/Planks
* Quartz Bricks

**Comment by LeadMagnet** [comment]
> Are these still in a testbranch? Cause the latest version via apt cant render chains and soulfire as well as blackstone walls

As of today this branch has not been merged

https://github.com/overviewer/Minecraft-Overviewer/pulls

I'm sure they plan on a merge shortly.

**Comment by RossosHEX** [comment]
> 
> 
> Added soul_fire, soul_lanterns and soul_torch
> 
> ![soul_lightning_1](https://user-images.githubusercontent.com/41533138/90974175-f3cd6c00-e528-11ea-9166-ada85ab9921e.png)
> 
> ![soul_lightning_2](https://user-images.githubusercontent.com/41533138/90974177-f9c34d00-e528-11ea-99a9-ab2fab33ddee.png)
> 
> I closed the old pull requests, since they are included in #1839.
> 
> To test this use:
> 
> ```shell
> git fetch origin pull/1839/head:testsoullight
> git checkout testsoullight
> ```
> 
> sha1 is [66ef67b](https://github.com/overviewer/Minecraft-Overviewer/commit/66ef67bb781de1894f3267dfdba6240c503f1095)

Tested your branch. (0.16.8 d23fdb1 )

These blocks rendered fine:
* Warped Planks
* Warped Slab
* Warped Stairs
* Polished_Blackstone_Bricks
* Polished_Blackstone_Bricks_Slab
* Polished_Blackstone_Bricks_Stairs

This blocks is not rendered:
* Quartz Bricks

Other minor thing is that the max zoom didn't render.

**Comment by IncredibleHolg** [comment]
new pull request #1856 

closed prevoius pull requests, since all changes are included 

![quartz_missing01](https://user-images.githubusercontent.com/41533138/95009653-ff31ae00-0623-11eb-8491-8f0488eb06b8.png)

![quartz_missing02](https://user-images.githubusercontent.com/41533138/95009657-03f66200-0624-11eb-8715-86d01d2bb52e.png)

sha1: 5d61f9a

**Comment by RossosHEX** [comment]
Thank you!

**Comment by IceBlueFire** [comment]
Not sure if this is the right place to post this but i'm noticing the following textures are missing in version 0.16.8 (731af71)

No rendering of:

Warped Planks
Warped Stairs
Warped Slab
Warped pressure plate
Warped Fence
Warped Trapdoor
Warped Fence Gate
Warped Door
Warped Sign

Crimson Planks
Crimson Stairs
Crimson Slab
Crimson pressure plate
Crimson Fence
Crimson Trapdoor
Crimson Fence Gate
Crimson Door
Crimson Sign

![download](https://user-images.githubusercontent.com/780366/97626851-804c4d00-19f8-11eb-90bb-2b144e5d6d70.png)

**Comment by IncredibleHolg** [comment]
@IceBlueFire 
Hi, 
this is already implemented, but not jet merged in the master branch.
If you have cloned this project, try this:
```bash
git fetch origin pull/1856/head:neweblocks
git checkout neweblocks
python3 setup.py build
```

**Comment by IncredibleHolg** [comment]
@CounterPillow 
Hi, I start to tose track of the branches for this issue, since it is hanging around for so long now.
Can we merge the #1856 and then create a new issue to reorganize  the ID numbers?

**Comment by MateoPeri** [comment]
Hi, how is this going? Is any help needed?

**Comment by Doregon** [comment]
@IncredibleHolg @dreamwraith I don't know how much this helps, but I wrote a new part for the docs section and merged both of your changes directly into the master branch of my fork. I will open a new pull request relating to this and keep everything tidy on my end as I work on other aspects of Overviewer. (Also thanks in the first place, I'm currently rendering a large world and your 1.16 block work helped me a lot!)

**Comment by dreamwraith** [comment]
> @IncredibleHolg @dreamwraith I don't know how much this helps, but I wrote a new part for the docs section and merged both of your changes directly into the master branch of my fork. I will open a new pull request relating to this and keep everything tidy on my end as I work on other aspects of Overviewer. (Also thanks in the first place, I'm currently rendering a large world and your 1.16 block work helped me a lot!)

My part was minimal, as I was just trying to get things working for my little server I run for my family at home. Real credit to @IncredibleHolg for the work of the updates themselves. Thanks for combining it all into a single PR @Doregon - Hopefully at some point some of the PR's are accepted. I am sure they have more important things in their life right now, but if @CounterPillow would like help reviewing and verifying and approving PR's, etc I would be willing to spare a few hours every week to help out.

**Comment by Doregon** [comment]
Yeah. Someone wrote that they were concerned about the pull requests not getting accepted but I understand and obviously when it's doable, it's okay. I was thinking about asking if they would allow more people to review and accept as well--present more opportunity for people's code to make it onto the master branch and potentially re-open past projects, like OIL, while lifting some of the weight off of their shoulder.

**Comment by CounterPillow** [comment]
Hi, sorry for the long wait.

so is everything contained in the quartz brick PR, or are there other PRs that need to be merged? Talking specifically about 1.16 stuff. If possible, please link to the original stuff, I can rebase/merge as needed myself.

**Comment by dreamwraith** [comment]
> Hi, sorry for the long wait.
> 
> so is everything contained in the quartz brick PR, or are there other PRs that need to be merged? Talking specifically about 1.16 stuff. If possible, please link to the original stuff, I can rebase/merge as needed myself.



I think this PR from Doregon is the one that ended up merging and including everything together with regards to the various missing blocks, etc

https://github.com/overviewer/Minecraft-Overviewer/pull/1901

**Comment by CounterPillow** [comment]
That one just seems to be IncredibleHolg's commits, plus some doc changes I don't agree with
